tests/fixtures/base-openapi.yml

```diff
@@ -104,6 +106,18 @@ components:
       name: Header-Param-From-Component
       schema:
         type: integer
+    HeaderB3TraceId:
+      in: header
+      name: X-B3-Traceid
```

Comment:
Is lower case important for the headers?

Reply:
We SHOULD use https://github.com/baloise-incubator/spectral-ruleset/blob/main/tests/132-SHOULD-prefer-hyphenated-pascal-case-for-HTTP-header-fields.test.ts but as it's a warn I check everything with lowercase in the rule so not break if someone ignores the warning.

Reply:
After Decision today we will stick to https://opensource.zalando.com/restful-api-guidelines/#132, so this will change to  X-B3-TraceID, X-B3-SpanID

Reply:
according to 132 will be X-B3-Traceid, X-B3-Spanid